src/drivers/distance_sensor/cm8jl65/cm8jl65.cpp

```diff
@@ -310,10 +300,10 @@ CM8JL65::ioctl(device::file_t *filp, int cmd, unsigned long arg)
 			default: {
 
 					/* convert hz to tick interval via microseconds */
-					int ticks = USEC2TICK(1000000 / arg);
+					int interval = (1000000 / arg);
```

Comment:
Some of the other drivers use `USEC_PER_SEC` instead of `1000000`, this might be a good opportunity to make that usage standard.